### PR TITLE
Retire deprecated close-stale plugin (now handled by lifecycle plugin)

### PR DESCRIPTION
**Pull request by fejta**
/assign @marun @cjwagner 

Migrate methods shared by `milestone-maintainer` and `close-stale` to `milestone-maintainer.go`
Delete `close-stale.go` munger.
Remove references to close-stale in mungegithub deployments and charts.

This functionality is now handled by the [lifecycle](https://github.com/kubernetes/test-infra/tree/master/prow/plugins/lifecycle) prow plugin.

ref https://github.com/kubernetes/test-infra/issues/5507

### CODE REVIEW

File: mungegithub/mungers/milestone-maintainer.go [hunk 1]
```diff
@@ -279,6 +281,130 @@ func durationGreaterThanZero(name string, value time.Duration) error {
...
+	return fmt.Sprintf("%d %s", days, dayString)
+}
+
+func durationToMinDays(duration time.Duration) string {
+	days := int(math.Floor(duration.Hours() / 24))
+	return dayPhrase(days)
+}
+
+func durationToMaxDays(duration time.Duration) string {
+	days := int(math.Floor(duration.Hours() / 24))
```
Comment (cjwagner) [inline mungegithub/mungers/milestone-maintainer.go]: s/Floor/Ceil/

File: mungegithub/mungers/milestone-maintainer.go [hunk 2]
```diff
@@ -279,6 +281,130 @@ func durationGreaterThanZero(name string, value time.Duration) error {
...
+
+		if lastInteresting.Before(*event.CreatedAt) {
+			lastInteresting = event.CreatedAt
+		}
+	}
+
+	return lastInteresting, true
+}
+
+func findLastModificationTime(obj *github.MungeObject) (*time.Time, bool) {
```
Comment (cjwagner) [inline mungegithub/mungers/milestone-maintainer.go]: All the `findLast*` functions in this file except for this one (`findLastModificationTIme`) were only used by the close-stale munger and can now be deleted.

**Comment by cjwagner** [comment]
One change, otherwise 
/lgtm

/hold

File: mungegithub/mungers/milestone-maintainer.go (hunk 1 above)
Comment (fejta) [reply mungegithub/mungers/milestone-maintainer.go]: @marun are you okay with me making that change?

**Pull request merged**